react/components/NumericStepper/index.js

```diff
@@ -213,15 +213,15 @@ class NumericStepper extends Component {
       <React.Fragment>
         {label && (
           <span
-            className={`numeric-stepper__label db mb3 w-100 c-on-base ${labelClasses}`}>
+            className={`vtex-numeric-stepper__label numeric-stepper__label db mb3 w-100 c-on-base ${labelClasses}`}>
             {label}
           </span>
         )}
-        <div className="numeric-stepper-container flex self-start">
+        <div className="vtex-numeric-stepper-container numeric-stepper-container flex self-start">
```

Comment:
what pattern is that used? BEM?

Reply:
🤷‍♂ Maybe, I just copied the class that was here before and added the prefix.